### PR TITLE
Update add-your-organization-brand-to-encrypted-messages.md

### PR DESCRIPTION
**Pull request by msbemba**
Added prerequisites for new-omeconfiguration (advanced message encyption)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7133

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder.

**Comment by yogkumgit** [comment]
@KCCross Kindly merge this PR. Thanks

File: microsoft-365/compliance/add-your-organization-brand-to-encrypted-messages.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ Use Windows PowerShell to modify one branding template at a time. If you have Ad
 
 ## Create an OME branding template (Advanced Message Encryption)
 
+Advanced Message Encryption is included in Microsoft 365 Enterprise E5, Office 365 E5, Microsoft 365 E5 (Nonprofit Staff Pricing), Office 365 Enterprise E5 (Nonprofit Staff Pricing), and Office 365 Education A5. If your organization has a subscription that does not include Office 365 Advanced Message Encryption, you can purchase it with the Microsoft 365 E5 Compliance SKU add-on for Microsoft 365 E3, Microsoft 365 E3 (Nonprofit Staff Pricing), or the Office 365 Advanced Compliance SKU add-on for Microsoft 365 E3, Microsoft 365 E3 (Nonprofit Staff Pricing), Office 365 SKUs, or the Microsoft 365 E5/A5 Information Protection and Governance SKU add-on for Microsoft 365 A3/E3.
```
Comment (KCCross) [inline microsoft-365/compliance/add-your-organization-brand-to-encrypted-messages.md:93]: Closing this PR as it's inaccurate. To add SKUs, look to the new Purview branding.

**Pull request closed without merging**